.php-cs-fixer.dist.php

```diff
@@ -106,7 +106,6 @@
         'fully_qualified_strict_types' => true,
         'function_declaration' => true,
         'function_to_constant' => true,
-        'function_typehint_space' => true,
```

Comment:
The corresponding alternative has already been configured, see

https://github.com/sebastianbergmann/phpunit/blob/c70b8108ac4cab82f67c66459ac74b3cdaef563e/.php-cs-fixer.dist.php#L330-L334